code/modules/atmospherics/machinery/components/fusion/hypertorus.dm

```diff
@@ -1021,87 +1021,87 @@
 		if(power_output)
 			switch(power_level)
 				if(1)
-					var/scaled_production = clamp(heat_output * 1e-2, 0, fuel_consumption) * delta_time
-					moderator_internal.gases[/datum/gas/carbon_dioxide][MOLES] += scaled_production * 0.95
-					moderator_internal.gases[/datum/gas/water_vapor][MOLES] += scaled_production *0.75
+					var/scaled_production = clamp(heat_output * 1e-2, 0, fuel_consumption)
+					moderator_internal.gases[/datum/gas/carbon_dioxide][MOLES] += scaled_production * 0.45
+					moderator_internal.gases[/datum/gas/water_vapor][MOLES] += scaled_production *0.35
 					if(m_plasma > 100)
 						moderator_internal.gases[/datum/gas/nitrous_oxide] += scaled_production * 0.5
-						moderator_internal.gases[/datum/gas/plasma][MOLES] -= min(moderator_internal.gases[/datum/gas/plasma][MOLES], scaled_production * 0.85)
+						moderator_internal.gases[/datum/gas/plasma][MOLES] -= min(moderator_internal.gases[/datum/gas/plasma][MOLES], scaled_production * 0.45)
 					if(m_bz > 150)
 						internal_output.assert_gases(/datum/gas/healium, /datum/gas/halon, /datum/gas/proto_nitrate)
-						internal_output.gases[/datum/gas/healium][MOLES] += scaled_production * 0.75
-						internal_output.gases[/datum/gas/halon][MOLES] += scaled_production * 0.55
-						internal_output.gases[/datum/gas/proto_nitrate][MOLES] += scaled_production * 0.25
-						moderator_internal.gases[/datum/gas/bz][MOLES] -= min(moderator_internal.gases[/datum/gas/bz][MOLES], scaled_production * 0.95)
+						internal_output.gases[/datum/gas/healium][MOLES] += scaled_production * 0.35
+						internal_output.gases[/datum/gas/halon][MOLES] += scaled_production * 0.15
+						internal_output.gases[/datum/gas/proto_nitrate][MOLES] += scaled_production * 0.05
+						moderator_internal.gases[/datum/gas/bz][MOLES] -= min(moderator_internal.gases[/datum/gas/bz][MOLES], scaled_production * 0.55)
```

Comment:
There's a lot of multiplier changes here, are you covering for the delta time removal?

Reply:
both covering the delta_time removal (see above for explanation on why) and to fix the infinite gas production (some of those gases were made with more moles than the consumed fuel and moderator)